incubator/traefik/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+name: traefik
+version: 2.0.0-alpha1
```

Comment:
chart version should not necessarily match app version.   Given this is in incubation, I would set `version: 0.0.1`

Reply:
The issue with that is that we already have `stable/traefik` that corresponds to the v1.X version of the app.

We wanted to have this chart be the v2 version of the chart to correspond with the v2 of the app.